libraries/botbuilder-dialogs-adaptive/src/steps/sendActivity.ts

```diff
@@ -46,19 +46,30 @@ export class SendActivity extends DialogCommand {
     constructor(activityOrText: Partial<Activity>|string, speak?: string, inputHint?: InputHints);
     constructor(activityOrText?: Partial<Activity>|string, speak?: string, inputHint?: InputHints) {
         super();
-        if (activityOrText) { this.activity.value = activityOrText }
-        if (speak) { this.activity.speak = speak }
-        this.activity.inputHint = inputHint || InputHints.AcceptingInput;
+        if (activityOrText) { this.activityProperty.value = activityOrText }
+        if (speak) { this.activityProperty.speak = speak }
+        this.activityProperty.inputHint = inputHint || InputHints.AcceptingInput;
     }
 
     protected onComputeID(): string {
-        return `send[${this.hashedLabel(this.activity.displayLabel)}]`;
+        return `send[${this.hashedLabel(this.activityProperty.displayLabel)}]`;
     }
 
     /**
      * Activity to send the user.
      */
-    public activity = new ActivityProperty();
+    private activityProperty = new ActivityProperty();
+
+    /**
+     * Public getter and setter for declarative activity configuration
+     */
+    public get activity(): Partial<Activity>|string {
```

Comment:
This was needed because the original implementation was constructor based, but we need property based setting of this to allow declarative patterns